modules/apps/27-interchain-accounts/depinject.go

```diff
@@ -34,19 +38,23 @@ func init() {
 type ModuleInputs struct {
 	depinject.In
 
-	Config        *modulev1.Module
-	Cdc           codec.Codec
+	// TODO: Config should define `controller_enabled` and `host_enabled` vars for keeper/module setup
+	Config *modulev1.Module
+	Cdc    codec.Codec
+
+	// TODO: runtime seems to expect that a module contains a single kvstore key.
 	ControllerKey *storetypes.KVStoreKey
 	HostKey       *storetypes.KVStoreKey
```

Comment:
ica uses two store keys which are passed to respective keepers (controllerkeeper and hostkeeper) below in `ProvideModule`.

Is there a way to handle multiple store keys for a module here? cc. @julienrbrt

Reply:
Mentioned a (hacky) alternative on Slack, but we will look into how many users have this use case.